src/main/java/com/denizenscript/depenizen/bukkit/objects/breweryx/BreweryRecipeTag.java

```diff
@@ -0,0 +1,367 @@
+package com.denizenscript.depenizen.bukkit.objects.breweryx;
+
+import com.denizenscript.denizencore.objects.Fetchable;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ColorTag;
+import com.denizenscript.denizencore.objects.core.DurationTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.core.ListTag;
+import com.denizenscript.denizencore.tags.Attribute;
+import com.denizenscript.denizencore.tags.ObjectTagProcessor;
+import com.denizenscript.denizencore.tags.TagContext;
+import com.dre.brewery.recipe.BEffect;
+import com.dre.brewery.recipe.BRecipe;
+import com.dre.brewery.recipe.RecipeItem;
+import com.dre.brewery.utility.Tuple;
+import org.bukkit.Color;
+
+public class BreweryRecipeTag implements ObjectTag {
+
+    // <--[ObjectType]
+    // @name BreweryRecipeTag
+    // @prefix breweryrecipe
+    // @base ElementTag
+    // @format
+    // The identity format for brewery is <recipe_name>
```

Comment:
Not sure what the naming for stuff in brewery's docs and such is, but wouldn't `recipe_id` make sure sense here?